distribution/lib/Standard/AWS/0.0.0-dev/package.yaml

```diff
@@ -8,3 +8,6 @@ authors:
 maintainers:
   - name: Enso Team
     email: contact@enso.org
+requires:
+  - Standard.Base
```

Comment:
I am running into issues.
- if `Standard.AWS` has its own **isolated layer**, then it complains that it cannot find `org.enso.base.file_system.FileSystemSPI`
- but if I connect `Standard.AWS` layer with `Standard.Base`, then it complains _Jackson_ is there twice!

It is hard to have both - _isolation and connection_ between the layers.

Addressed by 405e88de540cfa6ee9c7254724f853825e96bf50